gtsfm/scene_optimizer.py

```diff
@@ -174,6 +175,14 @@ def create_computation_graph(
         keypoints_graph_list = dask.delayed(lambda x, y: (x, y))(keypoints_graph_list, auxiliary_graph_list)[0]
         auxiliary_graph_list = []
 
+        # ensure cycle consistency in triplets
+        cycle_consistent_graph = dask.delayed(cycle_utils.filter_to_cycle_consistent_edges)(
+            i2Ri1_graph_dict, i2Ui1_graph_dict, two_view_reports_dict
+        )
+
```

Comment:
nit: this empty line is unnecessary. The next two lines are very much related to the previous.

Reply:
good point, updated now